src/BpmnModeler.ts

```diff
@@ -164,6 +164,10 @@ export class BpmnModeler implements vscode.CustomTextEditorProvider {
             extensionUri, 'dist', 'client', 'assets', 'bpmn-font', 'css', 'bpmn.css'
         ));
 
+        const waterMark = webview.asWebviewUri(vscode.Uri.joinPath(
```

Comment:
```suggestion
        const miranumLogo = webview.asWebviewUri(vscode.Uri.joinPath(
```
Damit es keine Missverständnisse gibt würde ich die Variable umbenennen.